isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/referenzarchitektur/inhalt.adoc

```diff
@@ -485,21 +485,22 @@ Aktuell wird diese Funktion vom Service-Gateway mit übernommen.
 [id="image-iam-integration-awl",reftext="{figure-caption} {counter:figures}"]
 image::referenzarchitektur/sicherheit_iam_integration_awl-2.dn.svg[]
 
-Bevor der Zugang zu der IsyFact-Anwendung gewährt wird, erfolgt in der Informations- und Dienstezone eine Zweifaktor-Berechtigungsprüfung:
+Bevor der Zugang zu der IsyFact-Anwendung gewährt wird, erfolgt in der Informations- und Dienstezone eine Berechtigungsprüfung:
 
-- Ein vorgeschalteter Loadbalancer (nicht im Bild) verlangt ein Client-Zertifikat des BVA/ITZ-Bund.
-Clients (Browser oder Anwendung), die nicht im Besitz dieses Client-Zertifikats sind, werden direkt abgewiesen.
--  Im Portal bzw. Authentication-Gateway wird die Identität des Nutzers geprüft (Authentifizierung).
-Der Nutzer muss dafür geheimes Wissen übermitteln (z.B. Name und Passwort für Anwender bzw. Client-ID und Secret für Anwendungen), das an einen zentralen Service (IAM-Service) in der Anwendungslandschaft weitergereicht wird.
+Im Portal bzw. Authentication-Gateway wird die Identität des Nutzers überprüft, indem Authentifizierungsdaten des Nutzers an einen zentralen Service (IAM-Service) in der Anwendungslandschaft weitergereicht werden.
 Ist der Nutzer dem IAM-Service bekannt, stellt er ein Access-Token aus, das auch die dem Nutzer zugeordneten Rollen enthält.
-
-Der Aufruf des Nutzers wird zusammen mit diesem Access-Token an die Interaktions- bzw. Service-Schicht der IsyFact-Fachanwendung weitergeleitet.
+Die Anfrage des Nutzers wird dann zusammen mit diesem Access-Token an die Interaktions- bzw. Service-Schicht der IsyFact-Fachanwendung weitergeleitet.
 Dort wird (über ein Rolle-Rechte Mapping) geprüft, ob der Nutzer das zur Ausführung des gewünschten Service notwendige Recht besitzt (Autorisierung).
 
 Eine detaillierte Beschreibung der erwähnten Systeme (Portal, Service-Gateway, Authentication-Gateway, IAM) und der Autorisierungs- und Authentifizierungsprotokolle OAuth2.0 und OpenID Connect findet sich in xref:isyfact-standards-doku:isy-security:konzept/master.adoc#inhalt[Konzept des Querschnitt-Bausteins Security].
 
 NOTE: IAM steht für xref:glossary:literaturextern:inhalt.adoc#litextern-identity_management[Identity and Access Management].
 
+Weitere Sicherheit bietet die zusätzliche Verwendung eines Client-Zertifikats:
```

Comment:
Wenn der Loadbalancer dieses Zertifikat prüft, dann stellt sich die Frage, welche Komponente im Load-Balancer diese Aufgabe übernimmt. Muss dazu die security-Komponente auf den Loadbalancer deployed werden?
Ich glaube dass das eher nicht der Fall ist. 
Somit beschreibst du hier in deinem neuen Kapitel eine Funktionalität, die außerhalb der Funktionalität der Security-Komponente liegt. Wenn wir diese Funktionalität hier wirklich beschreiben wollen, dann sollten wir dies in einem anderen Kapitel machen.